pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -105,6 +105,25 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 	}
 	logger.Infof("set user %q bucket max to %d", p.cephUserName, maxBuckets)
 
+	maxObjects := options.ObjectBucketClaim.Spec.AdditionalConfig["maxObjects"]
```

Comment:
Probably better to have a _getter_ function for these additionalConfig values.

Reply:
will address in next PR

Reply:
Done